### PR TITLE
Fixe calendar issues

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: apps/festival/festival/src/app/dashboard/event/list/list.component.ts [hunk 1]
```diff
@@ -8,13 +8,19 @@ import { Observable, combineLatest } from 'rxjs';
 import { filter, switchMap, startWith } from 'rxjs/operators';
 import { FormControl } from '@angular/forms';
 
+const typesLabel = {
```
Comment (phpgeekfr) [inline apps/festival/festival/src/app/dashboard/event/list/list.component.ts:11]: we may need to find the appropriate location to re-use this in the future

File: apps/festival/festival/src/app/marketplace/event/calendar/calendar.component.ts [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+import { Observable, combineLatest } from 'rxjs';
 import { EventService } from '@blockframes/event/+state/event.service';
 import { Event } from '@blockframes/event/+state';
 import { InvitationQuery } from '@blockframes/invitation/+state';
-import { map, switchMap } from 'rxjs/operators';
+import { map, switchMap, startWith } from 'rxjs/operators';
+import { FormControl } from '@angular/forms';
+import { EventTypes } from '@blockframes/event/+state/event.firestore';
+
+const typesLabel = {
```
Comment (phpgeekfr) [inline apps/festival/festival/src/app/marketplace/event/calendar/calendar.component.ts:10]: duplicate

File: libs/event/src/lib/components/screening-item/screening-item.component.html [hunk 3]
```diff
@@ -1,7 +1,9 @@
-<img fxHide.lt-sm [imgRef]="poster" placeholderAsset="empty_poster.webp" alt="Movie poster of the screening"/>
+<a matRipple [routerLink]="['/c/o/marketplace/title', movie.id]">
+  <img fxHide.lt-sm [imgRef]="poster" placeholderAsset="empty_poster.webp" alt="Movie poster of the screening"/>
+</a>
 <article fxLayout="column" fxLayoutAlign="space-between start">
   <div fxLayout="column" fxLayoutAlign="start start">
-    <h3><a [routerLink]="['../../../event/', screening.id]">{{ screening.title }}</a></h3>
+    <h3><a [routerLink]="['/c/o/marketplace/event/', screening.id]">{{ screening.title }}</a></h3>
```
Comment (phpgeekfr) [inline libs/event/src/lib/components/screening-item/screening-item.component.html:6]: can't you use relativ path ? so we don't use the word "markeplace" in lib folder ?

Comment (GrandSchtroumpf) [reply libs/event/src/lib/components/screening-item/screening-item.component.html:6]: No this component can be used in so many places. For now it's not an issue. If it becomes one we could create a pipe for that.

File: apps/festival/festival/src/app/marketplace/event/calendar/calendar.component.ts (hunk 2 above)
Comment (GrandSchtroumpf) [reply apps/festival/festival/src/app/marketplace/event/calendar/calendar.component.ts:10]: I'm following Laurent's rule here ^^
- 2 times: duplicate
- 3 times: factorize

File: apps/festival/festival/src/app/dashboard/event/list/list.component.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/festival/festival/src/app/dashboard/event/list/list.component.ts:11]: yep

File: apps/festival/festival/src/app/marketplace/event/calendar/calendar.component.ts (hunk 2 above)
Comment (phpgeekfr) [reply apps/festival/festival/src/app/marketplace/event/calendar/calendar.component.ts:10]: When will you factorize ?^^

Comment (GrandSchtroumpf) [reply apps/festival/festival/src/app/marketplace/event/calendar/calendar.component.ts:10]: When I've got a third case :)

**Pull request merged**